src/exercise/01.js

```diff
@@ -3,27 +3,27 @@
 
 import * as React from 'react'
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName}) {
+  const [name, setName] = React.useState(initialName);
+  const nameInput = React.useRef(null);
```

Comment:
bonus points for using fancy refs 🏆

Reply:
Didn't know how to use those until this course! Points to the course for that.